src/main/java/com/opinta/controller/DictionariesController.java

```diff
@@ -67,4 +73,10 @@ public DictionariesController(TariffGridService tariffGridService,
             return new ResponseEntity<>(NOT_FOUND);
         }
     }
+
+    @RequestMapping("/addresses")
+    public ResponseEntity<?> getAddressesByPostcode(@RequestParam(value="postcode") String postcode) {
+        List<CityDto> cities = cityService.getAllCitiesByPostcode(postcode);
```

Comment:
Please, change 
@RequestParam(value="postcode") String postcode
to
@RequestParam String postcode

The reason is if annotated argument name matches with real http request parameter name, Spring will discover and inject it even without explicit (value = "...") description.